rust_build_utils/rust_utils.py

```diff
@@ -26,6 +26,7 @@ class CargoConfig:
     arch: str
     debug: bool
     rust_target: str = ""
+    features: list = field(default_factory=list)
```

Comment:
how does that differ from:
```suggestion
    features: list = field(default_factory=list)
```